plugin/src/main/java/io/snyk/eclipse/plugin/views/snyktoolview/IssueTreeNode.java

```diff
@@ -19,7 +16,17 @@ public IssueTreeNode(Issue issue) {
 
 	@Override
 	public String getText() {
-		return getIssue().getDisplayTitle();
+		String displayTitle = getIssue().getDisplayTitle();
+
```

Comment:
I think it's better if this logic is in getDisplayTitle

Reply:
There is no getDisplayTitle function to override, or did I misunderstand what you are asking here?